giotto/diagrams/preprocessing.py

```diff
@@ -267,6 +272,7 @@ def inverse_transform(self, X, copy=None):
         return Xs
 
 
+@adapt_fit_transform_docs
 class Filtering(BaseEstimator, TransformerMixin):
```

Comment:
Agree about the weirdness of gerund, but I'd also avoid putting "Diagrams" in there for brevity and also because it is already a class in giotto.diagrams

Reply:
Fine with me, it was just a suggestion